test/WebSites/ActivatorWebSite/Components/ValueComponent.cs

```diff
@@ -1,19 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using Microsoft.AspNet.Mvc;
-
-namespace ActivatorWebSite
-{
-    [ViewComponent(Name = "Value")]
-    public class ValueComponent : ViewComponent
-    {
-        [Activate]
-        public ViewService MyViewService { get; set; }
```

Comment:
Do we have tests elsewhere that test constructor injection into a view component. Fine to file a follow-up bug if not. Idea is to cover the general scenario of using some service or other in a view component.